### PR TITLE
Add test cases

### LINKED ISSUE #63: Add test cases for PR#59 to hit 75% coverage

### PR DESCRIPTION
**Pull request by waihin26**
Fixes #63

### CODE REVIEW

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -141,6 +141,8 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
     private void assertCommandFailure(String inputCommand, Class<? extends Throwable> expectedException,
             String expectedMessage, Model expectedModel) {
         assertThrows(expectedException, expectedMessage, () -> logic.execute(inputCommand));
+        System.out.println(expectedModel.getFilteredPersonList().toString());
+        System.out.println(model.getFilteredPersonList().toString());
```
Comment (sumomomomomo) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:145]: Remove debug print statements

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public class PersonBuilder {
     public static final String DEFAULT_PHONE = "85355255";
     public static final String DEFAULT_EMAIL = "amy@gmail.com";
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
-    public static final String DEFAULT_FEES = "564";
+    public static final String DEFAULT_FEES = "100";
```
Comment (sumomomomomo) [inline src/test/java/seedu/address/testutil/PersonBuilder.java:26]: Why does this need to be changed? Can't see reason

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 1 above)
Comment (waihin26) [reply src/test/java/seedu/address/logic/LogicManagerTest.java]: fixed

File: src/test/java/seedu/address/testutil/PersonBuilder.java (hunk 2 above)
Comment (waihin26) [reply src/test/java/seedu/address/testutil/PersonBuilder.java]: Fixed

**Comment by hooyushing** [comment]
LGTM

**Pull request merged**